app/models/bag_man_request.rb

```diff
@@ -39,7 +39,7 @@ def cancel!(reason)
     unless cancelled
       transaction do
         update!(cancelled: true, cancel_reason: reason)
-        replication_transfer.cancel!(reason)
+        replication_transfer.cancel!(reason, last_error)
         send_cancel(replication_transfer)
       end
     end
```

Comment:
Where does the `last_error` come from?  What is the relationship, if any, between the `last_error` and the `reason`?  If this is not obvious, add commentary to explain the context.

Should the `send_cancel` method check whether the background jobs are enabled?